docker/dockcross-android-arm

```diff
@@ -179,7 +179,7 @@ fi
 #------------------------------------------------------------------------------
 # Now, finally, run the command in a container
 #
-docker run --rm \
+sudo docker run --rm \
```

Comment:
These scripts are generated automatically by dockcross scripts. And using `sudo` should be unnecessary for regular users. So don't change these lines.